.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -103,3 +101,11 @@ body:
         </details>
     validations:
       required: true
+
+  - type: checkboxes
+    id: validation
+    attributes:
+      label: Validation
+      options:
+        - label: Yes, I've included all information below (version, config, etc.).
```

I guess this should now be "above" if this will end up at the bottom?

```suggestion
        - label: Yes, I've included all information above (version, config, etc.).
```